classes/class-wc-connect-service-settings-store.php

```diff
@@ -103,13 +103,24 @@ public function get_origin_address() {
 			$wc_address_fields = array();
 			$wc_address_fields[ 'company' ] = get_bloginfo( 'name' );
 			$wc_address_fields[ 'name' ] = wp_get_current_user()->display_name;
-			$base_location = wc_get_base_location();
-			$wc_address_fields[ 'country' ] = $base_location[ 'country' ];
-			$wc_address_fields[ 'state' ] = $base_location[ 'state' ];
-			$wc_address_fields[ 'address' ] = '';
-			$wc_address_fields[ 'address_2' ] = '';
-			$wc_address_fields[ 'city' ] = '';
-			$wc_address_fields[ 'postcode' ] = '';
+
+			$wc_countries = WC()->countries;
+			if ( method_exists( $wc_countries, 'get_base_address' ) ) {
```

Comment:
Do you think this need a comment to explain the change is due to a new feature in WC 3.2?